Ihor/Netflix/src/HomePage/SearchPanel/SearchPanel.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import "./SearchPanel.css";
+import { parse } from "query-string";
+import * as H from "history";
+
+interface ISearchPanelProps {
+  handleSearchChange({
+    search,
+    searchBy,
+  }: {
+    search: string;
+    searchBy: string;
+  }): void;
+  location: H.Location;
+}
+
+interface ISearchPanelState {
+  value: string;
+  searchBy: string;
+}
+
+class SearchPanel extends React.Component<
+  ISearchPanelProps,
+  ISearchPanelState
+> {
+  state: ISearchPanelState = {
+    value: "",
+    searchBy: "title",
+  };
+
+  componentDidMount() {
+    const query = parse(this.props.location.search) as {
+      searchBy: string;
+      search: string;
+    };
+    let { searchBy, search } = query;
+    searchBy ? this.setState({ searchBy: searchBy }) : (searchBy = "title");
```

Comment:
(searchBy = "title") WHY ??  you set it to the local variable that you don't use